api/src/main/java/org/openmrs/util/PrivilegeConstants.java

```diff
@@ -880,4 +880,11 @@
 	
 	@AddOnStartup(description = "Able to manage the search index")
 	public static final String MANAGE_SEARCH_INDEX = "Manage Search Index";
+	
+	@AddOnStartup(description = "Able to get order sets")
+	public static final String GET_ORDER_SET = "Get Order Set";
```

Comment:
By convention this should be plural i.e GET_ORDER_SETS and 'Get Order Sets' for the value